spec/ruby/floating_points_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'floating_points'
+
+RSpec.describe FloatingPoints do
+  it 'parses test properly' do
+    r = FloatingPoints.from_file('src/floating_points.bin')
+    
+    delta = 1e-6
+
+    expect(r.single_value).to be_within(delta).of 1.2345
+    expect(r.single_value_be).to be_within(delta).of 1.2345
+    expect(r.double_value).to be_within(delta).of 123.456
+    expect(r.double_value_be).to be_within(delta).of 123.456
```

Comment:
These probably has to be exact matches. It's not like there are any machine-dependent or floating point precision-dependent in just reading the value in its serialized form.